package.json

```diff
@@ -7,7 +7,7 @@
     "url": "https://short-d.com/r/extcode"
   },
   "scripts": {
-    "build": "yarn prep && yarn cp:assets && envsub --env-file .env build/manifest.json && tsc",
+    "build": "yarn cp:assets && envsub --env CUR_GIT_TAG=`git describe --tags --abbrev=0` --env-file .env build/manifest.json && tsc",
```

Comment:
Can we get this from environmental variable so that we can pass it in through build script?